src/components/sendOTP/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next';
+
+import SignUpHeader from 'src/components/reusable/header';
+import OTPMessageField from 'src/components/sendOTP/OTPMessageField';
+import SuccessfulVerification from 'src/components/sendOTP/SuccessfulVerification';
+
+const AccountVerification = () :JSX.Element => {
+    const { t } = useTranslation();
+    const [isSubmitted, setIsSubmitted] = useState(false); 
```

Comment:
type

Reply:
![Screenshot from 2023-11-15 21-18-44](https://github.com/ZenBit-Tech/ctrlchamps_fe/assets/101258024/91ed9b46-889e-4602-97f0-8b2900c0e27b)
I don't understand what's wrong, for me it looks like I've got all needed types, provide me with extra info please

Reply:
done,  sorry for inconvenience